<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Starter</class>
 <widget class="QDialog" name="Starter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>650</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxModel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuration :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Working Folder :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditWorkingFolder">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditConfiguration">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonWorkingFolderChange">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>21</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfigurationChange">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>21</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxAnalysis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>671</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Analysis of the model for canonical experiments</string>
   </property>
   <widget class="QSpinBox" name="spinBoxCombToFit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFitOnlyComb">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>do only comb nr</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxIdentifOnlyParam">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>only param.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCombToIdentif">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxParamToIdentif">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton3CanoIdentifPerComb">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Identifiability for combination nr</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2CanoFitAround">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fitting around parameter set</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>20</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widgetParameterSet" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonParameterSetChange">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>21</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditParameterSet">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonUseParameterFromFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use parameter set from file :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonUseParameterDefault">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use default values as parameter set</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSimulations">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>671</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Simply Running a differentiation</string>
   </property>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxTimeSwitch">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCytokToTitrate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>hrs</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxConditionToSwitchFrom">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>condition</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxConditionForCytokTitration">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>condition</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>in</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>condition</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxConditionForCellDensity">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton6CanoDiffAB">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>with antibody combinations</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton6CanoDiffJ4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>With data from Exp J4</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxConditionFromOrTo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>From</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>To</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QRadioButton" name="radioButton9TitrCellDensity">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Titration of cell density in </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton6CanoDiff">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Canonical Differentiation (J5)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton7SwitchMedium">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Switching differentiation medium at t=</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton8TitrCytok">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Titration of cytokine doses</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton1CanoFitPerComb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fitting all combinations separately</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton4CanoIdentifAround">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Identifiability around parameter set</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton5Sensitiv">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensitivity around parameter set</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFittingAroundBestSet">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(followed by fittings around best set)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPleaseSelectParamSet">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>630</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select a parameter set file.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Multiexperiment</string>
   </property>
   <widget class="QRadioButton" name="radioButton10MultiObjective">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi-Objective optimization from all experiments</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Starter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Starter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
